packages/phrases/src/locales/en/translation/admin-console/upsell/paywall.ts

```diff
@@ -0,0 +1,50 @@
+const paywall = {
+  applications:
+    '{{count, number}} application of <planName/> limit reached. Upgrade plan to meet your team’s needs. For any assistance, feel free to <a>contact us</a>.',
+  applications_other:
+    '{{count, number}} applications of <planName/> limit reached. Upgrade plan to meet your team’s needs. For any assistance, feel free to <a>contact us</a>.',
+  machine_to_machine_feature:
+    'Upgrade to a paid plan to create machine-to-machine application, along with access to all the premium features. For any assistance, feel free to <a>contact us</a>.',
+  machine_to_machine:
+    '{{count, number}} machine-to-machine application of <planName/> limit reached. Upgrade plan to meet your team’s needs. For any assistance, feel free to <a>contact us</a>.',
+  machine_to_machine_other:
+    '{{count, number}} machine-to-machine applications of <planName/> limit reached. Upgrade plan to meet your team’s needs. For any assistance, feel free to <a>contact us</a>.',
+  resources:
+    '{{count, number}} API resource of <planName/> limit reached. Upgrade plan to meet your team’s needs. <a>Contact us</a> for any assistant.',
+  resources_other:
+    '{{count, number}} API resources of <planName/> limit reached. Upgrade plan to meet your team’s needs. <a>Contact us</a> for any assistant.',
+  scopes_per_resource:
+    '{{count, number}} permission per API resource of <planName/> limit reached. Upgrade now to expand. <a>Contact us</a> for any assistant.',
+  scopes_per_resource_other:
+    '{{count, number}} permissions per API resource of <planName/> limit reached. Upgrade now to expand. <a>Contact us</a> for any assistant.',
+  custom_domain:
+    'Unlock custom domain functionality and a range of premium benefits by upgrading to a paid plan. Don’t hesitate to <a>contact us</a> if you need any assistance.',
+  social_connectors:
+    '{{count, number}} social connector of <planName/> limit reached. To meet your team’s needs, upgrade plan for additional social connectors and the ability to create your own connectors using OIDC, OAuth 2.0, and SAML protocols. Feel free to <a>contact us</a> if you need any assistance.',
```

Comment:
Why the component here `<planName/>` starts with a lowercase letter? looks strange

Reply:
No idea. I was just copy-pasting from original phrases. Let me check

Reply:
<img width="442" alt="image" src="https://github.com/logto-io/logto/assets/12833674/a754874f-ed65-4088-b4ca-a842b2d993a9">

Seems to be an existing issue. Will open a new PR to rename it if you think it necessary